includes/ocos.h

```diff
@@ -60,6 +60,18 @@ struct OrtTensorDimensions : std::vector<int64_t> {
   bool IsScalar() const{
     return empty();
   }
+
+  bool IsVector() const{
```

Comment:
it's a little weird of this method that the vector can be 2d, though first dim is 1.

Reply:
Sure, currently,` IsVector` only supports `[n]` format.